content/vendor/guides/_index.md

```diff
@@ -26,6 +28,16 @@ These guides will assume you've completed one of the previous guides for either
 
 In this guide, we'll review best practices for integrating persistent stores like databases, queues, and caches. We'll explore ways to give your end user the option to either embed an instance with the application, or connect your application to an external cluster that they will manage. We'll use a PostgreSQL database as an example, configuring an example app to connect.
 
+###### [Airgapped Cluster](/vendor/guides/airgapped-cluster)
+
+The KOTS Airgapped Cluster is an advanced guide that shows how to install a KOTS app in an Airgapped environment.
+
+
+###### [CI/CD Integration](/vendor/guides/ci-cd-integration)
+
+The KOTS CI/CD Integration guide is an advanced workflow that allows you to use a similar workflow to [Kots Starter GitHub Actions](https://github.com/replicatedhq/replicated-starter-kots/blob/master/.github/workflows/main.yml#L21) but for your own internal git repository and CI/CD systems like GitHub, GitLab, CircleCI, TravisCI, Jenkins, etc.
```

Comment:
does this need to point to the `make release` line or?

Reply:
Good call, II might just remove the link entirely since this is just a summary.

```suggestion
The KOTS CI/CD Integration guide is an advanced workflow that allows you to use a similar workflow to KOTS Starter GitHub Actions but for your own internal git repository and CI/CD systems like GitHub, GitLab, CircleCI, TravisCI, Jenkins, etc.
```